Glyssen/Dialogs/VoiceActorAssignmentDlg.cs

```diff
@@ -39,9 +39,9 @@ public VoiceActorAssignmentDlg(Project project)
 
 				foreach (KeyValuePair<int, CharacterGroup> pair in charGroupTemplate.CharacterGroups)
 					characterGroups.Add(pair.Value);
-			}
 
-			m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
+				m_project.CharacterGroupList.PopulateEstimatedHours(m_project.IncludedBooks);
```

Comment:
Ah.  Didn't realize we were persisting that.  I guess in the case I saw in which it didn't populate, my character group file must have been created before the code was implemented.

Storing the characterIds is different because that is actually part of the definition of a group, but I don't see any reason not to store the estimated hours.